packages/clerk-js/src/ui/components/UserProfile/DeleteUserForm.tsx

```diff
@@ -35,8 +36,13 @@ export const DeleteUserForm = withCardStateProvider((props: DeleteUserFormProps)
         throw Error('user is not defined');
       }
 
+      // Store the session so it can be ended after the user is deleted
+      const session = client.lastActiveSessionId;
       await user.delete();
-      await navigateAfterSignOut();
+      await setActive({
+        session,
+        beforeEmit: navigateAfterSignOut,
+      });
```

Comment:
❓Why don't we do 
```suggestion
      await setActive({
        session: null,
        beforeEmit: navigateAfterSignOut,
      });
```
and update the code in clerk.ts to be ?
```tsx
    const shouldSignOutSession = newSession === null;
    if (shouldSignOutSession) {
      this.#broadcastSignOutEvent();
      eventBus.dispatch(events.TokenUpdate, { token: null });
    }
```

Reply:
I explicitly added the session just to handle multi-sessions